src/giskard_hub/resources/conversations.py

```diff
@@ -20,18 +25,19 @@ def create(
         dataset_id: str,
         messages: List[ChatMessage],
         demo_output: Optional[ChatMessage] = NOT_GIVEN,
-        tags: List[str] = NOT_GIVEN,
-        checks: List[TestCaseCheckConfig] = NOT_GIVEN,
+        tags: Optional[List[str]] = NOT_GIVEN,
+        checks: Optional[List[TestCaseCheckConfig]] = NOT_GIVEN,
```

Comment:
Can we make them defaulting to empty list instead ? just to make it easier for the user